apcd-cms/src/apps/submission_form/templates/submission_form/submission_form.test.html

```diff
@@ -0,0 +1,2965 @@
+      /* HACK: FP-1808: To hide field we want to use (until it can be used) */
+      div.field-wrapper.checkboxselectmultiple:not(.required) {
+        display: none;
+      }
+    </style>
+
+    <style id="form-css-request-submit--fp-1808">
+      /* HACK: FP-1808: To warn admin and user that this field is broken */
+      div.field-wrapper.checkboxselectmultiple.required > label::after {
+        display: block;
+        margin-top: 0.5em;
+
+        /* To mimic c-message */
+        /* https://github.com/TACC/tup-ui/blob/main/libs/core-styles/src/lib/_imports/components/c-message.css */
+        padding: 15px 20px;
+        border: var(--global-border--thick);
+        font-size: var(--global-font-size--small);
+      }
+      html[class*="cms-structure-mode-"]
+        div.field-wrapper.checkboxselectmultiple.required
+        > label::after {
+        content: "A multi checkbox field that is required is not well implemented. Do not require this field or replace it with multiple required Yes/No radio fields.";
+
+        /* To mimic c-message c-message-error */
+        /* https://github.com/TACC/tup-ui/blob/main/libs/core-styles/src/lib/_imports/components/c-message.css */
+        color: var(--global-color-danger--dark);
+        background-color: var(--global-color-danger--weak);
+        border-color: var(--global-color-danger--normal);
+      }
+      html:not([class*="cms-structure-mode-"])
+        div.field-wrapper.checkboxselectmultiple.required
+        > label::after {
+        content: "This field is not well implemented. Select all boxes to bypass the error with this field.";
+
+        /* To mimic c-message c-message-warning */
+        /* https://github.com/TACC/tup-ui/blob/main/libs/core-styles/src/lib/_imports/components/c-message.css */
+        color: var(--global-color-warning--dark);
+        background-color: var(--global-color-warning--weak);
+        border-color: var(--global-color-warning--normal);
+      }
+    </style>
+  </aside>
+
+  <div class="row">
+    <div class="col">
+      <h1>Request to Submit</h1>
+
+      <hr style="margin-top: 10px" />
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col">
+      <p style="margin-bottom: 30px">
+        This form should be completed and submitted to register as a data
+        submitter. Please review the
+        <a
+          href="https://sph.uth.edu/research/centers/chcd/apcd/techinical/TXAPCD-Data_Submission_Guide.pdf"
+          target="_blank"
+          >Data Submission Guide</a
+        >&nbsp;for details about completing and submitting this form, paying
+        special attention to the schedule of submissions including test files,
+        historical files, and monthly files.
+      </p>
+
+      <hr />
+
+      <!--suppress ALL -->
+
+      <div class="forms" id="">
+        <div class="description">
+          <h4>Organization</h4>
+        </div>
+
+        <div class="form-wrapper">
+          <form
+            action=""
```

Comment:
**Idea.** We may be able to just use `<form method="POST">` ([ref 1](https://stackoverflow.com/a/13520198/11817077), [ref 2](https://stackoverflow.com/a/16800092/11817077)). I'll test.